### ISSUE TITLE
Folder structure too complicated, difficult to synchronize archives

### ISSUE THREAD

**Issue by Gordon-Dry** [Grimm-Aerospace/GrimmModlets#2]
Digging into "Content", there is the list of folders, each single one with its own subfolder structure NAME\GameData\GrimmAerospace\GrimmModlets\NAME\

I suggest a change to Content\GameData\GrimmAerospace\GrimmModlets\NAME to avoid going back and forth again and again.

**Comment by Grimmas** [comment]
I see what you mean, but the reason for this is because each folder under Content is basically a separate mini-mod that people may decide to install (or not). I fear that if I organize it the way you describe, people will get confused about which folder they can delete or not. 

But I realize that this structure is also not ideal (in fact, in my first draft I had exactly the structure you describe) so I am open to discussing it.

**Comment by Gordon-Dry** [comment]
You could use PatchManager mod for it.
By default all disabled, user decides whose to use.
The ones that are already shipped with other mods should be removed or get a distinctive warning tag.

**Comment by Grimmas** [comment]
That would just lead to a stream of complaints along the line of "I installed it but nothing happened."

**Comment by Gordon-Dry** [comment]
Well, who uses PatchManager should know that he/she uses it and how to use it.
And a proper description is 80% of the experience.
I know, many talented but otherwise autistic programmers can code but are incapable of writing at least a minimum of documentation, wiki or user manual.
But that must not be the norm.
Also; you cannot do failsafe for any circumstance within 8 billion people with a given percentage of ignorance or dumbness.
There are people who understand well - and others without a clue. Let them be. And don't punish those who understand.

And - a wonky folder structure also triggers complaints. See this issue ;)

**Comment by Grimmas** [comment]
I'll think about it more, at the very least it might be an interesting weekend project.

**Comment by Grimmas** [comment]
So I looked into this and I only found two ways of implementing "conditional enable". Either by injecting C# code at runtime into ModuleManager (the way Kerbalism does it) or by creating lots of empty folders under GameData to trick ModuleManager into thinking they are mods (the way RO/RP-1/MechJeb do it). 

Neither approach makes sense in the context of this mod, but I'm also not very happy with the current solution either, so I'm going back to my previous structure, one where all modlets (except the redundant ones) are enabled by default and the user can delete the ones he doesn't want. As a consequence it should also be easier to navigate now.
